### PR TITLE
Add curious cases endpoint

### PR DESCRIPTION
**Pull request by leexgh**
Part of https://github.com/genome-nexus/genome-nexus/issues/549
image

### CODE REVIEW

File: service/src/main/java/org/cbioportal/genome_nexus/service/internal/CuriousCasesServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                String chr = splited[0];
+                String range = splited[1];
+                if (range.contains("-")) {
+                    String[] splitedRange = range.split("-");
+                    int start = Integer.parseInt(splitedRange[0]);
+                    int end = Integer.parseInt(splitedRange[1]);
+                    String comment = null;
+                    String pubmedIds = null;
+
+                    if (chr.equals("4") && start >= 55595380 && end <= 55595381) {
```
Comment (inodb) [inline service/src/main/java/org/cbioportal/genome_nexus/service/internal/CuriousCasesServiceImpl.java:56]: this should be any deletion event from `55593576` to `55593580`, see https://bit.ly/3ewk2hq

File: service/src/main/java/org/cbioportal/genome_nexus/service/internal/CuriousCasesServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                        comment = "Potential in-frame deletion event at intron10-exon 11 boundary, commmon in Gastrointestinal Stromal Tumors";
+                        pubmedIds = "c(15507676,27600498, 32697050)";
+                    }
+                    if (chr.equals("5") && start == 112151184 && end == 112151184) {
+                        comment = "Potential introduction of donor splice site, common in colorectal cancers";
+                        pubmedIds = "c(29316426)";
+                    }
+                    if (chr.equals("3") && start >= 41266015 && end <= 41266246) {
```
Comment (inodb) [inline service/src/main/java/org/cbioportal/genome_nexus/service/internal/CuriousCasesServiceImpl.java:64]: can prolly be from `41265579`

File: data/curiousCases/CuriousCases.tsv [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Gene 	Exon (not necessary)	Variant Type	Variant Classification	Overlap Enire Region 	SNP Position	SNP change	Comment	Pubmedids
```
Comment (leexgh) [inline data/curiousCases/CuriousCases.tsv:1]: Probably ok to delete this file

**Review by inodb — APPROVED** [review]
I think this is ok to merge for now, since it's mostly for a prototype

**Pull request merged**